packages/nextra-theme-docs/src/components/toc.tsx

```diff
@@ -110,7 +110,7 @@ export function TOC({ headings, filePath }: TOCProps): ReactElement {
           className={cn(
             hasHeadings &&
               'mt-8 border-t bg-white pt-8 shadow-[0_-12px_16px_white] dark:bg-dark dark:shadow-[0_-12px_16px_#111]',
-            'sticky bottom-0 pb-8 dark:border-neutral-800 flex flex-col gap-2',
+            'sticky bottom-0 pb-8 dark:border-neutral-800 flex flex-col items-start gap-2',
```

Comment:
toc links should not take full width 
<img width="306" alt="image" src="https://user-images.githubusercontent.com/7361780/188214887-ac147404-8b96-42ec-a83f-21d4e843232f.png">